### PR TITLE
Get actions collection on Preview Container update

### PR DESCRIPTION
**Pull request by michaelrogers**
Closes  #7158 

### Describe your changes:

Added updated lifecycle hook to fetch actions collection after the Preview Container is updated. This resolves not all menu items being available when viewing in Preview mode.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [ ] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [ ] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: src/ui/preview/PreviewContainer.vue [hunk 1]
```diff
@@ -108,6 +108,9 @@ export default {
       this.addExistingViewBackToParent();
     }
   },
+  updated() {
+    this.getActionsCollection(this.view);
+  },
```
Comment (ozyx) [inline src/ui/preview/PreviewContainer.vue:113]: This is an unconventional change and might raise some eyebrows in the future. We should add a `// FIXME:` and a brief note explaining why it's here. We should also reference the issue URL

**Review by ozyx — CHANGES_REQUESTED** [review]
LGTM, just one suggestion to add a comment. We should also add a quick e2e test

File: src/ui/preview/PreviewContainer.vue (hunk 1 above)
Comment (ozyx) [reply src/ui/preview/PreviewContainer.vue:113]: You know what? This totally escaped my mind. We might be able to just use `await nextTick()` in the `mounted()` hook before running `this.getActionsCollection(this.view);`. That way we're not running that on every update, even though I think it technically only gets run once anyway.

Comment (michaelrogers) [reply src/ui/preview/PreviewContainer.vue]: Unfortunately, nextTick didn't bail us out this time so I left the updated lifeycle hook and added a fixme to give more context.

Comment (ozyx) [reply src/ui/preview/PreviewContainer.vue]: Damn! Ok, sounds good

**Comment by ozyx** [comment]
@michaelrogers just need to run a `lint:fix` on this bad boy. the failing e2e test should pass once we update to master

**Review by ozyx — APPROVED** [review]
LGTM

**Pull request merged**